references/classification/sampler.py

```diff
@@ -15,7 +15,7 @@ class RASampler(torch.utils.data.Sampler):
     https://github.com/facebookresearch/deit/blob/main/samplers.py
     """
 
-    def __init__(self, dataset, num_replicas=None, rank=None, shuffle=True):
+    def __init__(self, dataset, num_replicas=None, rank=None, shuffle=True, seed=0):
```

Comment:
I agree this might be useful but it's also a paradigm we don't do on TorchVision. Don't get me wrong, this approach is popular with other libraries it's just that setting seeds on constructors is something we don't typically do.

@fmassa thoughts?

Reply:
@datumbox The RA sampler is modified from torch.utils.data.DistributedSampler which also has a parameter `seed`. The default value of `seed` is 0, so it does not affect the behavior of RA samper if users don't explicitly set a seed.

Reply:
Fair enough. Since DistributedSampler supports it we can also have it.